src/Calculations.ts

```diff
@@ -0,0 +1,8 @@
+import { Point3D } from './DataTypes';
+
+
+/* Compute the angle between the first three Point3Ds in the provided Array,
+ * in degrees rounded to the nearest tenth, with index 0 as theta. */
```

Comment:
I'm not sure the calculations module should be rounding, that seems more like a display thing

Reply:
Yeah, that makes sense. Fixed